### PR TITLE
Nios integration and unit tests for all remaining nios new modules

### PR DESCRIPTION
**Pull request by justjais**
##### SUMMARY

Modified the nios test container to support new added nios modules. New modules added:
nios_a_record
nios_aaaa_record
nios_cname_record
nios_mx_record
nios_naptr_record
nios_srv_record
nios_txt_record

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

nios

##### ANSIBLE VERSION

```
2.7
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: test/integration/targets/nios_a_record/defaults/main.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+testcase: "*"
+test_items: []
```
Comment (gundalow) [inline test/integration/targets/nios_a_record/defaults/main.yaml]: Missing newline

File: test/integration/targets/nios_a_record/meta/main.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+dependencies:
+  - prepare_nios_tests
```
Comment (gundalow) [inline test/integration/targets/nios_a_record/meta/main.yaml]: Missing newline

File: test/integration/targets/nios_aaaa_record/defaults/main.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+testcase: "*"
+test_items: []
```
Comment (gundalow) [inline test/integration/targets/nios_aaaa_record/defaults/main.yaml]: Missing newline

File: test/integration/targets/nios_aaaa_record/meta/main.yaml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+dependencies:
+  - prepare_nios_tests
```
Comment (gundalow) [inline test/integration/targets/nios_aaaa_record/meta/main.yaml]: Missing newline

**Review by gundalow — APPROVED** [review]
Looks good, just some minor comments about newlines missing

File: test/integration/targets/nios_a_record/defaults/main.yaml (hunk 1 above)
Comment (justjais) [reply test/integration/targets/nios_a_record/defaults/main.yaml]: Done

File: test/integration/targets/nios_a_record/meta/main.yaml (hunk 2 above)
Comment (justjais) [reply test/integration/targets/nios_a_record/meta/main.yaml]: Done

File: test/integration/targets/nios_aaaa_record/defaults/main.yaml (hunk 3 above)
Comment (justjais) [reply test/integration/targets/nios_aaaa_record/defaults/main.yaml]: Done

File: test/integration/targets/nios_aaaa_record/meta/main.yaml (hunk 4 above)
Comment (justjais) [reply test/integration/targets/nios_aaaa_record/meta/main.yaml]: Done

**Pull request merged**